assets/js/editor.js

```diff
@@ -47,7 +47,7 @@ wp.domReady( function() {
         return blockName === 'core/image' ?
             'nhsuk-image' :
             className;
-    }*/
```

Comment:
Line 57 below has another comment closing which is redundant and may cause issues